lib/lotus/model/migrator/adapter.rb

```diff
@@ -87,25 +87,28 @@ def load
         # @since 0.4.0
         # @api private
         def version
-          return unless @connection.tables.include?(MIGRATIONS_TABLE)
+          return unless connection.adapter_connection.tables.include?(MIGRATIONS_TABLE)
 
-          if record = @connection[MIGRATIONS_TABLE].order(MIGRATIONS_TABLE_VERSION_COLUMN).last
+          if record = connection.adapter_connection[MIGRATIONS_TABLE].order(MIGRATIONS_TABLE_VERSION_COLUMN).last
             record.fetch(MIGRATIONS_TABLE_VERSION_COLUMN).scan(/\A[\d]{14}/).first.to_s
           end
         end
 
         private
 
+        attr_reader :connection
+
         # @since 0.4.0
```

Comment:
because you modifed `new_connection` method please remove these two lines and add in the comments with `x.x.x`